app/buyers/views/work_orders.py

```diff
@@ -74,12 +74,12 @@ def create_new_work_order(framework_slug, lot_slug, brief_id):
         )['workOrder']
 
     except APIError as e:
+        form.errors['seller'] = [e.message]
```

Comment:
The way `_form_errors.html` is written, if I pass it just as a string each letter will be displayed as its own `<li>`. Wrapping in a list kept it to one line item